app/src/main/res/values-en/strings.xml

```diff
@@ -835,30 +835,30 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_crossing_type_marked">Marked</string>
     <string name="quest_crossing_type_unmarked">Without road markings</string>
 
-    <string name="quest_cycleway_title2">"Is there a cycleway here? What type?"</string>
-    <string name="quest_cycleway_resurvey_title">"Is this still the cycleway situation here?"</string>
-    <string name="quest_cycleway_value_track">cycle track</string>
-    <string name="quest_cycleway_value_lane">cycle lane</string>
-    <string name="quest_cycleway_value_lane_soft">cycle lane (dashed markings)</string>
+    <string name="quest_cycleway_title2">"Is there a bike lane or path here? What type?"</string>
+    <string name="quest_cycleway_resurvey_title">"Is this still the cycling situation here?"</string>
+    <string name="quest_cycleway_value_track">separated bike path</string>
+    <string name="quest_cycleway_value_lane">bike lane</string>
+    <string name="quest_cycleway_value_lane_soft">bike lane (dashed markings)</string>
```

Comment:
The dashed markings are a reference to a Vienna Convention marking, correct? Is there a need to internationalize this description? The U.S. uses dashed lines at “[mixing zones](https://nacto.org/publication/urban-bikeway-design-guide/intersection-treatments/through-bike-lanes/)” where cyclists generally stay within the lines but motor vehicles run rampant through them. That sounds much different than [the `cycleway=soft_lane` proposal](https://wiki.openstreetmap.org/wiki/Proposed_features/cycleway%3Dsoft_lane).

Reply:
> The U.S. uses dashed lines at “[mixing zones](https://nacto.org/publication/urban-bikeway-design-guide/intersection-treatments/through-bike-lanes/)” where cyclists generally stay within the lines but motor vehicles run rampant through them

Yes, that's pretty much the meaning of dashed line markings elsewhere too. `cycleway=lane + cycleway:lane=advisory` generally means 
- optional for cyclists to use (but cyclists are still obligated to keep right as always) 
- not exclusive to cyclists, other traffic may cross into the lane if necessary

Exact definition depend on legislation. For all intents and purposes, it is very similar to the failed soft_lane proposal. The creation of [`Key:cycleway:lane`](https://wiki.openstreetmap.org/wiki/Key:cycleway:lane) and StreetComplete's usage of these values was a response of that. I think there should be some links to how this page came about on that page.

Reply:
I guess it seemed different to me because mixing zones are by definition short and always flanked by or overlaid by a turn lane, whereas the photos in that proposal look more prolonged, like an alternative to a sharrow. The dashed lines in the MUTCD are analogous to [line extensions in an intersection](https://mutcd.fhwa.dot.gov/HTM/2009/part3/part3b.htm#section3B08).